server/application/application.go

```diff
@@ -66,6 +69,7 @@ func NewServer(
 		repoClientset: repoClientset,
 		appComparator: controller.NewAppStateManager(db, appclientset, repoClientset, namespace),
 		enf:           enf,
+		projectLock:   projectLock,
 	}
 }
 
```

Comment:
I don't see appRBACName() modified in this review. This should change to use project instead of repo

Reply:
Fixed